src/api/navigator/RoomSettingsErrorType.ts

```diff
@@ -0,0 +1,10 @@
+export class RoomSettingsErrorType
```

Comment:
You don't need to create these enums, they are already defined in `RoomSettingsSaveErrorParser` in the renderer